composer.json

```diff
@@ -14,6 +14,10 @@
         {
             "name":     "The contributors",
             "homepage": "http://github.com/knplabs/Gaufrette/contributors"
+        },
+        {
+            "name":     "Faustine Larmagna",
```

Comment:
This should also be removed. Contributors are not all listed individually here (it would be unmaintainable). This is why the previous author is linking to the list of contributors on github